### ISSUE TITLE
Why is our app icon not appearing in command bar?

### ISSUE THREAD

**Issue by catmanjan** [OfficeDev/microsoft-teams-library-js#1764]
We have an app deployed as an enterprise app via Teams administration

I notice that when I type @appname in the command bar it has a grey icon with 4 squares

It seems apps in the store have their icons appear in the same spot?

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/137350/c32b8594-75f3-4f1f-9fb1-8993277a9d68)

**Comment by Prasad-MSFT** [comment]
@catmanjan - We tried to repro the scenario with a deployed app and we can see the app icon being rendered correctly in command bar.
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/62237649/d5376954-b8b9-429a-82af-2474089ce480)

Checked in `Microsoft Teams Version 1.6.00.13566`

**Comment by catmanjan** [comment]
@Prasad-MSFT yes the icon appears after selecting the app in the search results, but if you click X what do you see?

**Comment by Prasad-MSFT** [comment]
Yes @catmanjan - Before selecting the app, the app icon doesn't show. It shows default icon only.
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/62237649/5d48544e-9826-4223-a607-efff5fc74593)

This seems to be by design behavior. However, we will confirm this once from engineering team and update here.

**Comment by Prasad-MSFT** [comment]
Hi @catmanjan - We got confirmation that this behaviour is by design. 
We recommend you give your feedback in [Teams Feedback Portal](https://feedbackportal.microsoft.com/feedback/forum/ad198462-1c1c-ec11-b6e7-0022481f8472).
